### PR TITLE
Proposed changes to [Quick Pick] intro/explanation text

### PR DESCRIPTION
**Pull request by avni**
Proposed changes to intro/explanation text and minor layout changes

### CODE REVIEW

**Comment by holta** [comment]
@avni I think we can (try!) to be much more concise?

And clean up some of the HTML formatting/indenting ideally?

What do you & others think of the following attempt to to reduce the wordiness, trying for a cleaner + much newbie-friendlier UX?

```
          <h2>Quick Pick</h2>
          <p>Your IIAB SD card must be <b>128GB</b> or larger (some require 256GB!) to install one of these content collections.
            If any content requires a particular service, it will be installed and enabled automatically if not already present.
            After the content collection is done installing, you might also want to add more/specific content, using other Install Content menu options.</p>
          <p></p>
          <p><b>Note:</b> If any content fails to download due to an Internet problem, please come back here after all jobs are completed, and try again.
            (Trying again should fix this relatively quickly, e.g. if your IIAB home page has broken links due to failed content downloads.)</p>
          <p></p>
          <p><b>Note:</b> Some content in these collections may have more restrictive licenses than you'd prefer.
             If so, select the specific content you need, using other Install Content menu options.
             Alternatively, you can remove content that was automatically installed.</p>
          <p></p>
          <p><b>To begin, please select a <b>Content Collection:</b></p>
```

PS Mentioning hard disks & SSD's & other particularities is a distraction that detracts.

(An effective UX uses "as few words as possible" in most all cases!)

**Comment by avni** [comment]
Thanks for the edits, @holta. Incorporating most of them. Hard drive more generic than SD card, yes - so using that instead. And I do think the first sentence provides valuable context for the reader, so leaving this: Quick Pick enables you to install a curated collection of content items that are commonly used together.

**Comment by holta** [comment]
My perspective is that the top 2 sentences contain a lot of (empty) hot air.

Personally I'd go with 1 single sentence that gets right to the point, something like...

```
To install one of these curated content collections, your IIAB SD card must be <b>128GB</b> or larger (some require 256GB!)
```

**Comment by holta** [comment]
> Note: Some content in these collections may have more restrictive licenses than you'd prefer.
If so, select the specific content you need, using other Install Content menu options.
Alternatively, you can remove content that was automatically installed.

I agree that the above content licensing disclaimer doesn't add much in its current position.

But:

Might there be a way to dramatically shrink this blurb &mdash; e.g. to make it appear as an aside, as a footnote at the bottom of the page, or perhaps as smallprint quite literally (and not just small, possibly an extremely small font?)

The reason I ask is that IIAB will eventually have to face Creative Commons (and similar) fresh/quality content ecosystem struggles head on.  As different new/free/open quality content models evolve and point the the way forward (if we want this learning movement to truly grow!)  A critical topic indeed, to address & engage those interested in priming the pump of global creativity/scraping here... that deserves a footnote... _Anyway Somewhere..._ possibly even a direct link to something we write up in http://FAQ.IIAB.IO or elsewhere?!

**Comment by avni** [comment]
> My perspective is that the top 2 sentences contain a lot of (empty) hot air.
> Personally I'd go with 1 single sentence that gets right to the point, something like...

> ```
> To install one of these curated content collections, your IIAB SD card must be <b>128GB</b> or larger (some require 256GB!)
> ```

I think they provide some valuable context and requirements to someone who's not familiar with admin-console and IIAB; this proposed revision does that too though - will update.

**Comment by avni** [comment]
> > Note: Some content in these collections may have more restrictive licenses than you'd prefer.
> > If so, select the specific content you need, using other Install Content menu options.
> > Alternatively, you can remove content that was automatically installed.
> 
> I agree that the above content licensing disclaimer doesn't add much in its current position.
> 
> But:
> 
> Might there be a way to dramatically shrink this blurb — e.g. to make it appear as an aside, as a footnote at the bottom of the page, or perhaps as smallprint quite literally (and not just small, possibly an extremely small font?)
> 
> The reason I ask is that IIAB will eventually have to face Creative Commons (and similar) fresh/quality content ecosystem struggles head on. As different new/free/open quality content models evolve and point the way forward (if we want this learning movement to truly grow!) A critical topic indeed, to address & engage those interested in priming the pump of global creativity/scraping here... that deserves a footnote... _Anyway Somewhere..._ possibly even a direct link to something we write up in http://FAQ.IIAB.IO or elsewhere?!

I had opted to leave this in as a footnote to the narrative, but removed when it wasn't in the first proposed edit of the PR and the meta feedback was around trying to make the intro text more concise. Would prefer to keep in somewhere as well. Will add back in.

**Comment by tim-moody** [comment]
>valuable context and requirements to someone who's not familiar with admin-console and IIAB

Please put in the preamble to the menu option (here) only a reminder of the caveats.

Please put material that provides context and further explanation of this menu item in the help.

Please put material aimed at those who are not familiar with Admin Console and IIAB in the documentation.

**Comment by avni** [comment]
Thanks @tim-moody  - I've added back in some preamble text/explanation and re-added the licence note as a footer. Are there other specific changes you would like to see?

**Comment by holta** [comment]
> your IIAB hard drive must be <b>128GB</b> or larger

The Raspberry Pi Foundation does not talk of "hard drives" and nor should we.

If we want to help out newcomers whose SD cards are too small, we should speak their language.

That probably means saying "SD cards" very directly, to avoid driving newcomers nuts with generalities.

Or, if we must speak in generalities (e.g. if we VM's really might become an important segment of our IIAB community in the coming year?) then we should say "storage."

File: roles/console/files/htmlf/40-install_content.html [hunk 1]
```diff
@@ -56,18 +56,14 @@ <h2>Get Started</h2>
...
-          <p><b>Note</b> that some content in these collections may have more restrictive licenses than you would prefer.</p>
-          <p>In that case you should select the content that meets your requirements using other menu options.</p>
-          <p>It is also possible to remove items that have been automatically installed.</p>
-          <p><b>Note</b> if content fails to download due an Internet problem, please come back here after all jobs are complete, and try again.</p>
-          <p>Trying again should quickly fill in the gaps (e.g. if your IIAB home page contains broken links due to failed content downloads).</p>
-          <p>Please select one of the following <b>Content Collections:</b></p>
-
+          <p>To install one of these curated content collections<sup><b>*</b></sup>, your IIAB hard drive must be <b>128GB</b> or larger (some require 256GB!).
+            If any content requires a particular service, it will be installed and enabled automatically if not already present.
+            After the content collection is done installing, you may also want to add more/specific content, using other Install Content menu options.</p>
```
Comment (holta) [inline roles/console/files/htmlf/40-install_content.html:61]:
```suggestion
            After the content collection is done installing, you can add more/specific content, using other Install Content menu options.</p>
```

**Review by holta — APPROVED** [review]

**Comment by avni** [comment]
> I had opted to leave this in as a footnote to the narrative, but removed when it wasn't in the first proposed edit of the PR and the meta feedback was around trying to make the intro text more concise. Would prefer to keep in somewhere as well. Will add back in.

Just went back through and realized that  the first proposed PR did have the license text in.. Not sure why I didn't see that when I was first looking at the diff! Apologies. The text is included in the latest version.

**Pull request merged**